documentation/modules/managing/proc-cluster-recovery-volume.adoc

```diff
@@ -195,7 +195,7 @@ If you include the Topic Operator in the deployment, the Topic Operator will del
 
 . Delete the internal topic store topics from the Kafka cluster:
 +
-[source,shell]
+[source,shell,subs="+quotes,attributes"]
```

Comment:
```suggestion
[source,shell,subs="+attributes"]
```

Reply:
Othewise, with the underscores on the topic names half of the command is in italics